packages/web/src/about/Team.tsx

```diff
@@ -117,64 +107,75 @@ const Portrait = React.memo(function _Portrait({
           >
             {purpose}
           </Text>
-          <View style={[standardStyles.row, standardStyles.elementalMarginTop]}>
+          <View style={standardStyles.row}>
             <Text style={[fonts.p, textStyles.heavy, styles.name]}>{name}</Text>
             {url && (
-              <a href={url}>
+              <a href={externalize(url)} target="_blank">
                 <External size={12} color={colors.dark} />
               </a>
             )}
           </View>
 
           <Text style={fonts.p}>
-            {company} – {team}
+            {company.trim()}
+            {team && `, ${team.trim()}`}
           </Text>
         </View>
       </Responsive>
     </>
   )
 })
 
+function externalize(url: string) {
+  try {
+    const uri = new URL(url)
+    return uri.href
+  } catch {
+    return `//${url}`
```

Comment:
will new URL(`//${url}`) work? seems nice to be consistent

Reply:
<img width="1013" alt="Screen Shot 2020-01-07 at 1 59 46 PM" src="https://user-images.githubusercontent.com/3814795/71933011-fc1d9600-3155-11ea-8770-ac8ee527bb6c.png">

doesn't quiet have the behavior i'm looking for.